### PR TITLE
Fix for more words named table

### PR DESCRIPTION
**Pull request by strstensky**

### CODE REVIEW

File: src/Console/Commands/GenerateControllerCommand.php [hunk 1]
```diff
@@ -8,91 +8,125 @@
...
+    {
+        return [];
+    }
+
+    /**
+     * @param String $name
+     * @return String
+     */
+    public function getSingularName(String $name) : String{
+        return collect(explode('_',str_singular($name)))->map(function ($word){
```
Comment (RichardDominik) [inline src/Console/Commands/GenerateControllerCommand.php]: Add static for the function

File: src/Console/Commands/GenerateControllerCommand.php [hunk 2]
```diff
@@ -8,91 +8,125 @@
...
+            return ucfirst($word);
+        })->implode('');
+    }
+
+    /**
+     * @param String $name
+     * @return String
+     */
+    public function getPluralName(String $name) : String{
+        return collect(explode('_',str_plural($name)))->map(function ($word){
```
Comment (RichardDominik) [inline src/Console/Commands/GenerateControllerCommand.php]: Add static for the function

File: src/Console/Commands/GenerateOrderColumnMigrationCommand.php [hunk 3]
```diff
@@ -49,6 +49,16 @@ protected function getDefaultNamespace($rootNamespace)
         return $rootNamespace . '\Http\Controllers\Admin';
     }
 
+    /**
+     * @param String $name
+     * @return String
+     */
+    public function getFileName(String $name) : String{
+        return collect(explode('_',str_plural($name)))->map(function ($word){
```
Comment (RichardDominik) [inline src/Console/Commands/GenerateOrderColumnMigrationCommand.php]: Add static for the function

**Review by branikclimbs — APPROVED** [review]

File: src/Console/Commands/GenerateControllerCommand.php (hunk 1 above)
Comment (branikclimbs) [reply src/Console/Commands/GenerateControllerCommand.php]: for what reason ?

**Comment by strstensky** [comment]
What about this forgotten PR? I would like to start using this package. Do we considerate the missing static as a reason not to merge? Is it even that important?

**Comment by dejwCake** [comment]
> What about this forgotten PR? I would like to start using this package. Do we considerate the missing static as a reason not to merge? Is it even that important?

Is it so hard to make that change?

**Review by dejwCake — APPROVED** [review]

**Pull request merged**